astacus/coordinator/plugins/clickhouse/plugin.py

```diff
@@ -83,6 +84,21 @@ class ClickHousePlugin(CoordinatorPlugin):
     max_concurrent_sync: int = 100
     use_system_unfreeze: bool = True
 
+    @root_validator(pre=True)
+    def _apply_max_concurrent_query_defaults(cls, values):  # pylint: disable=E0213
+        default_value = 10 * len(values["clickhouse"].nodes) if "clickhouse" in values else 100
```

Comment:
I'm not sure we should have complex conditional defaults, given that we can afford to deprecate&ignore old parameters, I'd rather just move to `_per_node`  concurrency limit for all limits.

Reply:
Thanks! I added _per_node and changed the code in steps - would it be better?